extensions/interactions/ItemSelectionInput/validator.js

```diff
@@ -89,16 +93,58 @@ oppia.filter('oppiaInteractiveItemSelectionInputValidator', [
         baseInteractionValidationService.getAnswerGroupWarnings(
           answerGroups, stateName));
 
-      if (!defaultOutcome ||
-          $filter('isOutcomeConfusing')(defaultOutcome, stateName)) {
-        warningsList.push({
-          type: WARNING_TYPES.ERROR,
-          message: (
-            'Please clarify the default outcome so it is less confusing to ' +
-            'the user.')
+      var selectedChoices = [];
+      if (maxAllowedCount === 1) {
+        seenChoices.forEach(function(seenChoice, choiceIndex) {
+          answerGroups.forEach(function(answerGroup) {
+            var ruleSpecs = answerGroup.rule_specs;
+            ruleSpecs.forEach(function(ruleSpec) {
+              var ruleInputs = ruleSpec.inputs.x;
+              ruleInputs.forEach(function(ruleInput) {
+                if (ruleInput === seenChoice) {
+                  if (ruleSpec.rule_type === 'Equals') {
+                    handledAnswers[choiceIndex] = true;
+                    if (ruleInputs.length > 1) {
+                      warningsList.push({
+                        type: WARNING_TYPES.ERROR,
+                        message: (
+                          'Please ensure that you only tick one answer ' +
```

Comment:
How about: `in answer group #NUM, rule #NUM, please select only one answer choice`?